VAMobile/src/screens/PaymentsScreen/PaymentHistory/PaymentDetailsScreen/PaymentDetailsScreen.tsx

```diff
@@ -54,29 +55,27 @@ function PaymentDetailsScreen({ navigation, route }: PaymentDetailsScreenProps)
           <Box accessibilityRole="header" accessible={true} mb={standardMarginBetween}>
             <TextView variant="MobileBodyBold">{paymentType}</TextView>
           </Box>
-          <TextView variant="MobileBodyBold" accessibilityRole="header" selectable={true}>
+          <TextView variant="MobileBodyBold" selectable={true}>
             {t('paymentDetails.amount')}
           </TextView>
           <TextView variant="MobileBody" selectable={true} mb={standardMarginBetween}>
             {amount}
           </TextView>
-          <TextView variant="MobileBodyBold" accessibilityRole="header" selectable={true}>
+          <TextView variant="MobileBodyBold" selectable={true}>
             {t('paymentDetails.method')}
           </TextView>
           <TextView variant="MobileBody" selectable={true}>
             {paymentMethod}
           </TextView>
           {isDirectDeposit && (
             <>
-              <TextView variant="MobileBodyBold" accessibilityRole="header" mt={standardMarginBetween}>
```

Comment:
I don't think the header role should be removed here right?

Reply:
Fixed